parsec/core/gui/app.py

```diff
@@ -215,8 +215,8 @@ def kill_window(*args):
         # QTimer wakes up the event loop periodically which allows us to close
         # the window even when it is in background.
         timer = QTimer()
-        timer.start(1000 if diagnose else 400)
-        timer.timeout.connect(kill_window if diagnose else lambda: None)
+        timer.start(400)
+        timer.timeout.connect(lambda: None)
```

Comment:
Any reason for removing `diagnose`?

Reply:
diagnose is already handled in the next lines, I didn't see the need to add it here (we could also remove the `if diagnose` stuff and keep this timer multi-purposed)